### DISCUSSION TITLE
ELF concentrations of large molecules using MoBi?

### DISCUSSION THREAD

**Discussion by RobinM92** [Open-Systems-Pharmacology/Forum#927] [Questions & Problems] [answered]
Dear community,

Another question from my side regarding macromolecules and the lung. In a different project I am trying to predict how much of a macromolecule ends up in the lung after systemic administration. Using the macromolecule model in PKSim works very well to predict the plasma concentrations after IV administration, and also the total lung concentrations coincide with lung homogenate data that I have (preclinical species). However, in some follow-up experiments BAL samples will be taken, which to my knowledge sample the epithelial lining fluid. Sadly, ELF is not (yet?) accessible as a container in PKSim, so it is difficult to corroborate PKSim predicitons to these data. Therefore, I thought to use MoBi again, but am a bit stuck on what the best approach is here. As I would need to expand the model to include ELF, two options seemed plausible for me. 

1) Would be to export the macromolecule IV model I already have in PKSim to MoBi and then add an ELF compartment to the lung manually, using parameter values from the recently published lung absorption model by Johanna Eriksson et al. However, I am not at all experienced in adding new tissue parts and am afraid I will miss a process and therefore wrongly predict the ELF concentrations; is there any material or tutorial on how to easily expand the spatial structures in MoBi?

2) Would be to start from the abovementioned lung model and expand it to accomodate large molecules; this would mean adding the endosome and FcRn recycling pathways. I tried to load in the large molecule standard file on top of this lung absorption MoBi file, but this was not succesfull so far and I again feel some tutorial on how to implement large molecules in MoBi would be very beneficial.

My question is thus which of these two paths would be the most straightforward and most scientifically sound to follow; in essence I want to 'merge' the macromolecule model and the lung absorption model into one, which in theory should be feasible but in practice I feel I am missing the finesse to connect the correct processes.

Thank you very much for you insights!
Kind regards,
Robin

**Comment by Erik-Sjogren** [comment, marked as answer]
Hi Robin,
I would suggest that you choose option 1, i.e., get inspiration from the Johanna et al model in terms of physiological structure and parameters and add this to your already established IV model. 
If you are not planning to simulate any pulmonary administration this should be quite feasible. However, depending on your molecule you may need to add some translocation processes etc. depending on how you anticipate your molecule to reach ELF from blood.
Good luck
Erik
